pay_ir/api/client.py

```diff
@@ -24,15 +24,15 @@ def __init__(self, api_key):
         self.api_key = api_key
 
 
-    def init_transaction(self, amount, redirect_url, factor_number=None):
+    def init_transaction(self, amount, redirect_url, factor_number=None, mobile=None, description=None):
         """
             Sends an initial request to 'API_URL_SEND' to retrieve a
                 transaction id. Sent data contains these variables:
 
                     - api (string): You API key for pay.ir
                     - amount (integer): Transaction amount in rials, should be greater than 1000.
                     - redirect (string): Urlencoded return address registered in pay.ir panel.
-                    - factorNumber (string): (optional).
+                    - factor_number (string): (optional).
```

Comment:
Please add mobile and description to function docstring.

Reply:
Done

Reply:
@moeenz is there anything else I can do about this PR?

Reply:
Merged.
Sorry for the delay.

Reply:
Thanks! It's alright.